.github/workflows/workflow.yml

```diff
@@ -4,8 +4,14 @@ on:
   push:
     branches:
       - main
+  pull_request:
   workflow_dispatch: # allows manual triggering
 
+# Prevent concurrent runs of this workflow.
```

Comment:
Hey @getvictor, does this mean that if two contributor open a PR around the same time, only one contributor will get feedback from the dry run? (errors)

Sorry for the delayed response.

Reply:
No, it should wait for the first dry run to finish before starting the other one. So, both contributors should get feedback.

This concurrency constraint was meant to prevent multiple real runs from running in parallel.